pc/src/menu.rs

```diff
@@ -1,26 +1,39 @@
-use std::vec;
-
 use crate::{update::init_level, Activity, GameState, Phase, LEVELS};
+use m3_models::{Key, ToPcGameEvent};
 
 use macroquad::{
 	hash,
 	prelude::*,
-	ui::{root_ui, widgets::Button, Skin}
+	ui::{root_ui, Skin}
 };
 
 const BUTTON_FONT_SIZE: u16 = 16;
 
-/*fn get_menu_skin() -> Skin {
+fn get_menu_skin() -> Skin {
 	{
 		let window_style = root_ui()
 			.style_builder()
 			.background(Image::from_file_with_format(
-				include_bytes!("../assets/img/Menu/selection_box_background.png"),
+				include_bytes!("../assets/img/Menu/menu_background.png"),
 				None
 			))
 			.background_margin(RectOffset::new(20.0, 20.0, 10.0, 10.0))
 			.margin(RectOffset::new(-20.0, -30.0, 0.0, 0.0))
 			.build();
+		Skin {
+			window_style,
+			..root_ui().default_skin()
+		}
+	}
+}
+
+fn get_button_skin() -> Skin {
+	{
```

Comment:
I don't think this extra brace that contains the entire function code serves any purpose